Ginger/GingerCoreNET/ValueExpressionLib/ValueExpression.cs

```diff
@@ -560,7 +560,7 @@ public void ReplaceDataSource(string p)
                     }
                     if (dt.Rows.Count == 0)
                     {
-                        mValueCalculated = "No Row found with " + Query;
+                        mValueCalculated = "No row found with selected query";
```

Comment:
please explain what was the issue with showing the Query in the error message?

Reply:
Query in error message doesnt look good so need to change the message as stated in bug

Reply:
i think we should keep the Query, only change error to "No row found with query: 'Select...'"

user should see the composed query so it will be easy for him to understand what is wrong in the query in case some parts of the query are dynamic